specification/06-arch_overview.adoc

```diff
@@ -27,12 +27,13 @@ and registering IOMMUs and PCIe root ports, and then binding physical devices
 interfaces (TDI) and TVMs together.
 
 With the CoVE-IO ABIs and flows, TDIs can access TVM confidential memory
-directly. Based on the TVM configuration, the confidential DMA memory
-could be all TVM memory or a subset of the TVM memory.
-CoVE-IO uses the Smmtt I/O MTT extension and the platform IOMMUs
-security domain specific Register Programming Interfaces (RPI) to grant TDIs
-with direct access to their bound TVM confidential memory and isolate it from
-DMA originating from any unbound TDI.
+directly through DMA. Based on the TVM configuration, the DMA accessible
+confidential memory could be all TVM memory or a subset of the TVM memory.
+A platform IOMMU instance must be tied to security domains holding TVMs to
```

Comment:
A platform may have multiple IOMMU instances.

Reply:
Yes. I meant that each TSM domain should hold one of those instances. I tried to clarify that statement now.